.github/pull_request_template.md

```diff
@@ -14,9 +14,15 @@ change me!
 - [ ] CHANGELOG is updated **OR** update is not needed
 - [ ] [documentation PR](https://spaces.redhat.com/display/StackRox/Submitting+a+User+Documentation+Pull+Request) is created and is linked above **OR** is not needed
 
-### Testing
+### Quality assurance
 
-- [ ] inspected CI results
+<!--
+General Availability requirements: https://github.com/stackrox/stackrox/blob/master/PR_GA.md
+Feature Flags usage: https://github.com/stackrox/stackrox/blob/master/pkg/features/README.md
+-->
+
+- [ ] the change is production ready: the feature is GA or otherwise gated by a feature flag
```

Comment:
Yes, to the spirit of the message but I have doubts about the form.
"Feature" and "change" aren't always equivalent. There are changes that are features even if authors don't recognize that first and there are changes that aren't features no matter how you stretch it. I'm concerned how this sentence reads for that latter class of changes.

How about tweaking it this way:
```suggestion
- [ ] the change is production ready: the change is GA or otherwise the functionality is gated by a feature flag
```